Frontend/src/register/components/registration_list.jsx

```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react'
+import deleteRegistration from '../../api/registration/delete/delete_registration'
+import getRegistrations from '../../api/registration/get/get_registrations'
+import updateRegistration from '../../api/registration/patch/update_registration'
+import styles from './list.module.scss'
+
+function StatusDropdown({ status, setStatus }) {
+  const options = ['waiting', 'accepted']
+  return (
+    <select onChange={(e) => setStatus(e.target.value)} value={status}>
+      {options.map((opt) => (
+        <option key={opt}>
+          {opt}
+        </option>
+      ))}
+    </select>
+  )
+}
+
+function RegistrationRow({
+  competitorId,
+  eventIDs,
+  serverStatus,
+  competitionID,
+  setRegistrationList,
+  registrationList,
+}) {
+  const [status, setStatus] = useState(serverStatus)
+
+  return (
+    <tr>
+      <td>{competitorId}</td>
+      <td>{eventIDs.join(',')}</td>
+      <td>
+        <StatusDropdown status={status} setStatus={setStatus} />
+      </td>
+      <td>
+        <button
+          onClick={(_) => {
+            updateRegistration(competitorId, competitionID, status)
+          }}
+        >
+          {' '}
+          Apply
+        </button>
+      </td>
+      <td>
+        <button
+          onClick={(_) => {
+            deleteRegistration(competitorId, competitionID)
+            setRegistrationList(
+              registrationList.filter((r) => r.competitor_id !== competitorId)
+            )
+          }}
+        >
+          {' '}
+          Delete
+        </button>
+      </td>
+    </tr>
+  )
+}
+
+export default function RegistrationList() {
+  const [competitionID, setCompetitionID] = useState('HessenOpen2023')
+  const [registrationList, setRegistrationList] = useState([])
+  return (
+    <div className={styles.list}>
+      <button
+        onClick={async (_) =>
+          setRegistrationList(await getRegistrations(competitionID))
+        }
+      >
+        {' '}
```

Comment:
You've got a lot of `{' '}` throughout all the components, I assume that was the linting and they aren't actually needed.